tests/integrations/minter/test_nominal_cases.cairo

```diff
@@ -0,0 +1,131 @@
+# SPDX-License-Identifier: MIT
+# Carbonable smart contracts written in Cairo v0.1.0 (test_nominal_case.cairo)
+
+%lang starknet
+
+# Starkware dependencies
+from starkware.cairo.common.bool import TRUE, FALSE
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+
+# Project dependencies
+from tests.integrations.minter.library import (
+    setup,
+    admin_instance as admin,
+    anyone_instance as anyone,
+)
+
+@view
+func __setup__{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}():
+    # GIVEN a deployed user contracts
```

Comment:
Gerkin allow to have several GIVEN statements on a single scenario, but it could be confusing.
In our case, it helps to split context per contracts imo, sounds good?